components/esp32_camera.rst

```diff
@@ -173,6 +173,30 @@ Test Setting:
     Camera uses PWM timer #1. If you need PWM (via the ``ledc`` platform) you need to manually specify
     a channel there (with the ``channel: 2``  parameter)
 
+Base64 encode/decode
+--------------------
+There are a helper functions which can decode or encode raw image data to base64 string. So it can be sent over MQTT or other protocols.
```

Comment:
```suggestion
There are helper functions available which can encode or decode raw image data into Base64 strings and vice versa. This can be useful to sent images over MQTT or other protocols.
```